phpunit.xml.dist

```diff
@@ -0,0 +1,35 @@
+<phpunit
```

Comment:
Adding in the full [XML references](https://github.com/GaryJones/plugin-boilerplate/blob/bd9ca02455f5fc2f3b57ade0459d672b9472d392/phpunit.xml.dist#L1-L3) (update URL to the right version), allows IDEs to pull the XSD and check the XML in this file is valid.